p2panda-js/package.json

```diff
@@ -10,8 +10,8 @@
     "lint": "eslint --ext .ts .",
     "test": "run-p --race test:mock-node test:run",
     "test:prepare": "run-s --silent wasm webpack:node",
-    "test:mocha": "mocha -r ts-node/register -r tsconfig-paths/register -r @babel/register 'src/**/*.test.ts'",
-    "test:run": "run-s clear test:prepare test:mocha",
+    "test:jest": "jest",
+    "test:run": "run-s clear test:prepare test:jest",
```

Comment:
I think we don't need `test:prepare` anymore or am I missing something? I was able to replace this line with

```
"test:run": "run-s clear wasm:node test:jest"
```

and then I just had to edit two lines in other files where type definitions where imported from `wasm-web` and point them at `wasm-node` instead. We can drop the whole `webpack` build and `wasm:web` as well.

Reply:
Hmn .. this would still break the tests when running them for the first time (you still need the wasm build).

Reply:
I've brought that change back to where it was .. there is surely some space to optimize the wasm build. This build step is only necessary a) when the repo got freshly cloned and b) when the `p2panda-rs` code changed. Currently I don't see an elegant way to only run the scripts in these cases, maybe with some smart content hash comparison?

Reply:
Ah, I think that was a misunderstanding. I'll make the change and then you can have a look, hope that's okay.

Reply:
I had another error yesterday that my head wouldn't really want to be wrapped around so I didn't push anything. We can do this later, yes.